proposals/relaxed-dead-code-validation/Overview.md

```diff
@@ -145,42 +149,36 @@ Note that no attempt is made to make this efficient or cleverly reduce line-coun
 
 ```javascript
 function validate(opcode) =
-  let reachable = !ctrls[0].unreachable
   switch (opcode)
     case (i32.add)
-      if (reachable)
-        pop_opd(I32)
-        pop_opd(I32)
-        push_opd(I32)
+      pop_opd(I32)
+      pop_opd(I32)
+      push_opd(I32)
 
     case (drop)
-      if (reachable)
-        pop_opd()
+      pop_opd()
```

Comment:
I believe this will spuriously error for the code fragment

```
unreachable
drop
```

Reply:
Ah, nice catch!